### PR TITLE
Fix docker --OODT_HOME-- issue

### PR DESCRIPTION
**Pull request by danlamanna**
Running docker (off of an image built from master) yields the following error:

```
Cannot find --OODT_HOME--/bin/setclasspath.sh
This file is needed to run this program
```

It seems as though a new file (env.sh) was added with the logic of replacing `--OODT_HOME--` with the actual path after the Dockerfile was last updated.

The way this was handled before this PR was by adding a copy of each file to DOCKER/ and manually find/replacing and running a COPY in the Dockerfile.

This PR fixes this issue in a more permanent manner by using sed on any configuration files in the checkout. As a result it removes the files that were manually duplicated and replaced from the repository.

I would appreciate feedback as this is my first time looking at/setting up ImageCat.

### CODE REVIEW

File: DOCKER/Dockerfile [hunk 1]
```diff
@@ -11,13 +11,12 @@ RUN apt-get update && \
...
 RUN pip install solrpy
 RUN conda install -c blaze apache-maven
-#RUN pip install xmlrpc }
 
 ENV JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
 ENV JRE_HOME=/usr/lib/jvm/java-7-openjdk-amd64
 
 RUN mkdir /deploy
-RUN git clone https://github.com/chrismattmann/imagecat.git
+RUN git clone -b fix-docker-oodt-home-replace-issue https://github.com/danlamanna/imagecat.git
```
Comment (danlamanna) [inline DOCKER/Dockerfile]: I will switch this back once it is ready to merge into master.

**Comment by chrismattmann** [comment]
@danlamanna seems like quite a bit of changes here? for example the sed is great, but why the rest of the removals (e.g., copying of solr, etc.?)

**Comment by danlamanna** [comment]
I'll add back the first COPY of `solr.xml` because I still can't track down how/where it's being used. Though it's difficult to test because Docker on master is currently broken.

Otherwise, the files removed are identical to the files that will exist after the sed is run, this prevents having to copy the files with `--OODT_HOME--` into the docker directory and add a line to the Dockerfile for each one.

**Comment by chrismattmann** [comment]
@danlamanna what about removing the *env.sh files - why remove them?

**Comment by chrismattmann** [comment]
@danlamanna can you update this PR I'd like to merge it thanks.

**Comment by chrismattmann** [comment]
ping @danlamanna

**Comment by chrismattmann** [comment]
ping @jeffbaumes

**Comment by jeffbaumes** [comment]
The `.sh` files should not be needed because they will exist in the Docker container through another means - this branch takes out logic that required them. Exactly what this "other means" is and ensuring this functions identically to before this branch is not easy because Docker is not working at all on master.

To say with certainty that the state is the same as before would take a good chunk of @danlamanna's time to spin back up on this, so for now we can close out this issue. Dan has a branch on his fork that works for him so it's ok if we don't upstream this now.

**Comment by chrismattmann** [comment]
if this is working on dan's fork...i'm good with it. Merging! :) 👍  thanks @jeffbaumes and @danlamanna

**Pull request merged**